EIPS/eip-7691.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 7691
+title: Blob throughput increase
+description: Increase the number of blobs to reach a new target of x blobs per block, to be defined by performing tests.
+author: Parithosh Jayanthi (@parithosh), Toni Wahrstätter (@nerolation), Sam Calder-Mason (@samcm), Andrew Davis (@savid)
+discussions-to: https://ethereum-magicians.org/t/eip-7691-blob-throughput-increase/19694
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-04-17
+---
+
+## Abstract
+
+This EIP proposes to increase to the number of blobs in a block to provide more scale to Ethereum via L2 solution that rely on L1 data capacity.
+
+## Motivation
+
+Ethereum, with its rollup centric roadmap, scales by relying on L2. Since the Dencun fork, the blob gas target and maximum was set to 3/6 respectively. The blob gas limit was arrived at based on a series of big block tests performed on the Ethereum mainnet network as well as a series of testnets. The values were chosen cautiously, as its extremely hard to predict the exact p2p behaviour of Ethereum mainnet.
+
+As we now have the Dencun upgrade live, we are able to use monitoring tools to check the network health. Initial monitoring indicates that we have a stable network with the current gas target, allowing us to consider starting a series of big block and blob tests to determine the theoretical headroom we currently have. The EIP would then be updated to include a proposed new blob gas target and limit.
```

Comment:
Could we add more details of the monitoring results or link to somewhere where it is documented? That would give the reader a clear picture of how much the network health has been affected and the fact that it is in an acceptable range.

Reply:
Yup, we will have multiple writeups published. We're currently working on some tools to get us more low level p2p visibility, once that's considered stable-ish we will trigger a new round of big block tests on Mainnet to gather the data.